### ISSUE TITLE
Compile Issue

### ISSUE THREAD

**Issue by RogueMaster** [leedave/flipper-zero-fap-boilerplate#2]
Getting a compile issue for flipper-zero-fap-boilerplate/scenes/boilerplate_scene_start.c on line 22 w/ "BoilerplateSceneStart"

Do you see that too, or is it just me somehow?

**Comment by leedave** [comment]
You're absolutely correct. My git repo is not in the same place I tested the code in. The file boilerplate_scene_start.c was renamed to boilerplate_scene_menu.c to avoid confusion with boilerplate_scene_startscreen.c. The old file should have been removed, done that now. 

Btw. let me know it would be a help to you if I send pull-requests for app updates in flipperzero-firmware-wPlugins. 
(I wouldn't recommend adding boilerplate there, as it is just a starting point for new apps. But I'm thinking of updating color_guess with the concept used here).
